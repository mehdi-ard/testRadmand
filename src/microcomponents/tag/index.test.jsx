import React from 'react';
import renderer from 'react-test-renderer';
import Tag from './';

it('renders default', () => {
  expect(renderer.create(<Tag>success</Tag>).toJSON()).toMatchSnapshot();
});

it('renders color success', () => {
  expect(
    renderer.create(<Tag color="success">success</Tag>).toJSON()
  ).toMatchSnapshot();
});

it('renders color processing', () => {
  expect(
    renderer.create(<Tag color="processing">processing</Tag>).toJSON()
  ).toMatchSnapshot();
});

it('renders color error', () => {
  expect(
    renderer.create(<Tag color="error">error</Tag>).toJSON()
  ).toMatchSnapshot();
});

it('color default', () => {
  expect(
    renderer.create(<Tag color="default">default</Tag>).toJSON()
  ).toMatchSnapshot();
});

it('color warning', () => {
  expect(
    renderer.create(<Tag color="warning">warning</Tag>).toJSON()
  ).toMatchSnapshot();
});

it('custom color', () => {
  expect(
    renderer.create(<Tag color="#f50">custom color</Tag>).toJSON()
  ).toMatchSnapshot();
});
