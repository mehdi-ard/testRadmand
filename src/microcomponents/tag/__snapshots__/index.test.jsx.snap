// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`color default 1`] = `
<span
  className="ant-tag ant-tag-default"
  style={
    Object {
      "backgroundColor": undefined,
    }
  }
>
  default
</span>
`;

exports[`color warning 1`] = `
<span
  className="ant-tag ant-tag-warning"
  style={
    Object {
      "backgroundColor": undefined,
    }
  }
>
  warning
</span>
`;

exports[`custom color 1`] = `
<span
  className="ant-tag ant-tag-has-color"
  style={
    Object {
      "backgroundColor": "#f50",
    }
  }
>
  custom color
</span>
`;

exports[`renders color error 1`] = `
<span
  className="ant-tag ant-tag-error"
  style={
    Object {
      "backgroundColor": undefined,
    }
  }
>
  error
</span>
`;

exports[`renders color processing 1`] = `
<span
  className="ant-tag ant-tag-processing"
  style={
    Object {
      "backgroundColor": undefined,
    }
  }
>
  processing
</span>
`;

exports[`renders color success 1`] = `
<span
  className="ant-tag ant-tag-success"
  style={
    Object {
      "backgroundColor": undefined,
    }
  }
>
  success
</span>
`;

exports[`renders default 1`] = `
<span
  className="ant-tag"
  style={
    Object {
      "backgroundColor": undefined,
    }
  }
>
  success
</span>
`;
